_posts/2017-10-06-DevopsDays-London.md

```diff
@@ -0,0 +1,217 @@
+inability to cope with ‘big data’ workloads – instead a combination of
+Mesos and Marathon was recommended for this.
+
+What can we take from this? Undoubtedly Kubernetes is an exciting
+prospect, but one that is still in its infancy. It was telling how few
+are currently using this for production workloads – perhaps ‘boring is
+powerful’ after all. Which leads us neatly on to…
+
+Boring is powerful
+==================
+
+Jon Topper from The Scale Factory gave a great talk on idea of keeping
+your platform ‘boring’, which can be split into two main ideas; first of
+all, that software is rarely complete right from the start, features are
+always added over time. As the ‘fashionableness’ of a product decreases
+over time, feature availability, quality, reliability, security and
+collective knowledge all increase. The release schedule of former
+industry darling MongoDB illustrates the point:
+
+![MongoDB release schedule](/images/MongoDB_release_schedule.png)
+
+For example - MongoDB did not get data journaling until it was 18 months
+old…! But plenty of organizations adopted it long before that, and
+usually put measures in place to work around missing features. Which
+increased operational overhead, engineer fatigue, and in turn reduced
+confidence in the product. This really highlights how business context
+and considerations are always required, but not always met, when making
+design and deployment decisions.
+
+So, as the maturity of a product increases, and the community around it
+develops, it becomes ‘boring’. And the more ‘boring’ you have within
+your platform, the more secure and reliable it will be. This is a good
+thing, and it should be celebrated! Especially for the on-call engineers
+who can now enjoy a good night’s sleep.
+
+The second main point of the talk was the contradiction that sometimes,
+yes, you will need to adopt new technologies. The speaker put forward
+the idea of ‘innovation tokens’, which similar to the SRE-style ‘error
+budget’ mechanism, allow a specific number of new projects per quarter,
+as long as they meet certain criteria; the project must be beneficial to
+you, as a key differentiator for your business. The project must be
+risk-worthy, and likely to give you a competitive advantage. The idea
+being that fewer products are adopted on the whim of one lead engineer,
+who having read a blog about the shiniest new app, immediately puts said
+app straight into production!
+
+Not jumping aboard the latest technology bandwagon is something we are
+already aware of – with the size of our platform, and the number of
+transactions we process at peak times, stability is our number one
+priority and this is helped when our engineers have in-depth knowledge
+of a product. There is much to be said for repeatable deployment
+patterns using familiar software packages!
+
+Jon’s full talk is available
+[here](https://www.youtube.com/watch?v=90EUfSpk96M&index=12&list=PLuEbc43fHqLhrfE7IOt9z3bcmofPhwu7H).
```

Comment:
'here' as a link is bad for accessibility reasons as it is hard to navigate links, make it more descriptive

Reply:
consider embedding the video too if possible